cms/users/tests/test_signal_handlers.py

```diff
@@ -132,20 +132,14 @@ def setUpTestData(cls) -> None:
 
     def test_user_profile_notifications_disabled_on_creation(self):
         self.assertFalse(self.profile.submitted_notifications)
-        self.assertFalse(self.profile.approved_notifications)
-        self.assertFalse(self.profile.rejected_notifications)
-        self.assertFalse(self.profile.updated_comments_notifications)
+        self.assertTrue(self.profile.approved_notifications)
```

Comment:
NP: As this test function is called `test_user_profile_notifications_disabled_on_creation` should we add a comment above explaining which ones are actually disabled and which ones are enabled, based on your PR description, just for posterity?